src/components/spinner/Spinner.tsx

```diff
@@ -57,39 +56,36 @@ export const Spinner = forwardRef<HTMLSpanElement, SpinnerProps>((props, ref) =>
 
   return (
     <>
-      <SpinnerWrapper className={wrapperClassName} inline={isInline}>
-        <SpinnerInner
-          {...radixProps}
-          aria-live="polite"
-          className={classNames({
-            it__loading: isLoading,
-          })}
+      <SpinnerInner
+        {...radixProps}
+        aria-live="polite"
+        className={classNames(wrapperClassName, {
+          it__loading: isLoading,
+        })}
+        /** Needs to also be inline if a visible label is present next to the Spinner icon */
+        inline={isInline || isDefined(label)}
+        isLoading={isLoading}
+        ref={ref}>
+        <SpinnerStyled
+          className={className}
           /** Needs to also be inline if a visible label is present next to the Spinner icon */
           inline={isInline || isDefined(label)}
-          isLoading={isLoading}
-          ref={ref}
-          role="status">
```

Comment:
Dropping the role on purpose? That's probably gonna break a lot of tests in the webapp repo

Reply:
The tests `getByRole('status')` with no issue. I changed the SpinnerInner to be an `output` tag, so its semantically correct (instead of a `<span role="status"/>`)